TS_src/client/src/App.tsx

```diff
@@ -24,17 +24,6 @@ function App() {
 		setActiveComponent(componentName);
 	};
 
-	// hook for tracking userTrip data
-	interface Trip {
-		active: boolean;
-		start_timestamp: string;
-		start_lat: string;
-		start_lng: string;
-		sos_timestamp: string;
-		sos_lat: string;
-		sos_lng: string;
-	}
-
 	interface TripProps {
```

Comment:
Could we also abstract the `TripProps` interface out? Even though it is only used here, I think it makes sense to just have all our types in one place and never have them in the frontend code that's run.

Reply:
Extracted TripProps and migrated to Types.tsx. Will need to update all other components in later PRs for best practice.